coralproject/talk
# Suspend user

## What does this PR do?
- Implements the new Suspend User Story [#144459753](https://www.pivotaltracker.com/story/show/144459753)
- Implements a [notification service](https://github.com/coralproject/talk/pull/600/files#diff-539b42b1f378342ce606737100fab9e0)


## How do I test this PR?

- Suspend any user in the mod queue
- Check if email has been received
- Login as the suspended user, see suspension note, and all mutations should fail.
- Wait until suspension time is over, and see if suspension has been lifted :-D
